src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -1068,16 +1068,20 @@ public void checkout(Run<?, ?> build, Launcher launcher, FilePath workspace, Tas
 
         try {
           checkoutCommand.execute();
-        } catch(GitLockFailedException e) {
+        } catch (GitLockFailedException e) {
```

Comment:
Ah, wait, this is one that slipped past me where IntelliJ did fix it and I failed to catch it, oops. I'll unfix it.